src/cmake/version.rc.in

```diff
@@ -0,0 +1,37 @@
+1 VERSIONINFO
+FILEVERSION     @ALICEVISION_INSTALL_VERSION_MAJOR@,@ALICEVISION_INSTALL_VERSION_MINOR@,@ALICEVISION_INSTALL_VERSION_REVISION@,0
+PRODUCTVERSION  @ALICEVISION_INSTALL_VERSION_MAJOR@,@ALICEVISION_INSTALL_VERSION_MINOR@,@ALICEVISION_INSTALL_VERSION_REVISION@,0
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904E4"
+        BEGIN
+            VALUE "CompanyName",      "AliceVision Organization\0"
+#if @ALICEVISION_INSTALL_LIBRARY@
+            VALUE "FileDescription",  "@ALICEVISION_INSTALL_NAME@ - AliceVision with @CMAKE_CXX_COMPILER_ID@ @CMAKE_CXX_COMPILER_VERSION@\0"
+#else // software
+            VALUE "FileDescription",  "@ALICEVISION_INSTALL_NAME@ - AliceVision @ALICEVISION_VERSION_MAJOR@.@ALICEVISION_VERSION_MINOR@.@ALICEVISION_VERSION_REVISION@\0"
+#endif
+            VALUE "FileVersion",      "@ALICEVISION_INSTALL_VERSION_MAJOR@.@ALICEVISION_INSTALL_VERSION_MINOR@.@ALICEVISION_INSTALL_VERSION_REVISION@.0\0"
+            VALUE "InternalName",     "@ALICEVISION_INSTALL_NAME@\0"
+            VALUE "LegalCopyright",   "Copyright (c) 2018 AliceVision contributors\0"
+            VALUE "ProductName",      "AliceVision Library | @ALICEVISION_INSTALL_NAME@\0"
+            VALUE "ProductVersion",   "@ALICEVISION_INSTALL_VERSION_MAJOR@.@ALICEVISION_INSTALL_VERSION_MINOR@.@ALICEVISION_INSTALL_VERSION_REVISION@.0\0"
```

Comment:
It is superfluous to duplicate the same version string in both FileVersion and ProductVersion. It'd be better to use ProductVersion to store git commit number instead.